types/canvasjs/index.d.ts

```diff
@@ -1,992 +1,1690 @@
-// Type definitions for CanvasJS v1.5.1
+// Type definitions for CanvasJS 1.9
 // Project: http://canvasjs.com/
-// Definitions by: Mark Overholt <https://github.com/mover5>
+// Definitions by: ShuYin Zhang  <https://github.com/brutalimp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace CanvasJS {
+export namespace CanvasJS {
```

Comment:
If this library isn't exposed as a module at all, you shouldn't use `export`, because that declares the shape of a module.
The way to declare a global library is `declare namespace`, like it was before.